templates/server-acl-init-clusterrole.yaml

```diff
@@ -22,6 +22,16 @@ rules:
     verbs:
       - create
       - get
+  - apiGroups: ["apps"]
+    resources:
+      - statefulsets
+    verbs:
+      - get
+  - apiGroups: [""]
+    resources:
+      - services
+    verbs:
+      - get
```

Comment:
I totally don't remember adding this so I'll delete it. Good catch. But it's interesting that we actually do make one call for Services:
```
			kubeSvc, err = c.clientset.CoreV1().Services("default").Get("kubernetes", metav1.GetOptions{})
```

However that must have been working before. Looking that code now, I'm not sure why it doesn't just use `kubernetes.default.svc.cluster.local` instead of the IP address (which is the whole reason it's calling this in the first place).

Reply:
Yep, it's calling it inside the `configureConnectInject`, but the helm chart already [adds](https://github.com/hashicorp/consul-helm/blob/24d3115a2f04e8b5058f43b1d67afe6eff20df19/templates/server-acl-init-clusterrole.yaml#L25-L35) `services` in case connect is enabled.